src/main/resources/api.yaml

```diff
@@ -207,3 +312,39 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+  ProjectMemberDto:
+    type: object
+    properties:
+      userId:
+        type: integer
+        format: int64
+      userDetails:
+        $ref: '#/definitions/UserDetailsDto'
+      role:
+        $ref: '#/definitions/UserRoleDto'
+  UserRoleDto:
+    type: string
+    enum:
+      - admin
+      - member
```

Comment:
Postulat z frontu:
`admin` -> `ADMIN`
`member` -> `MEMBER`

I to samo przy okazji przy `SprintState`, jakbyś mógł (znaczy można osobne issue, ALE PO CO)

(@szbrnl)

Reply:
również proszę